tests/functional/g/globals.py

```diff
@@ -44,12 +44,12 @@ def global_with_import_from():
 
 def global_no_assign():
     """Not assigning anything to the global makes 'global' superfluous"""
-    global CONSTANT  # [global-variable-not-assigned]
+    global CONSTANT  # [global-without-modification]
     print(CONSTANT)
 
 
```

Comment:
```suggestion

RAN_DB_SET = set()
RAN_DB_DICT= dict()

def init_connection_state(alias):
   """Initialize the database connection settings."""
    global RAN_DB_SET 
    global RAN_DB_DICT
    RAN_DB_SET.add(alias)
    return RAN_DB_DICT.set_default("color", "Palomino")
```

Reply:
Just so I understand - do you think the Pylint _code_ needs to change or the message _text_?

In this example the set and dict don't need global in order to be updated, right?
```python
(venv38) Marks-MacBook-Air-2:programming markbyrne$ python x.py 
{'color': 'Palomino'}
{'xxx'}
(venv38) Marks-MacBook-Air-2:programming markbyrne$ cat x.py 
RAN_DB_SET = set()
RAN_DB_DICT= dict()
def init_connection_state(alias):
    RAN_DB_SET.add(alias)
    return RAN_DB_DICT.update(color = "Palomino")


init_connection_state("xxx")
print(RAN_DB_DICT)
print(RAN_DB_SET)
```

Reply:
Nice point, actually the message name was better before, and there's nothing to handle: the global keyword is only really required for assignment only. I think the documentation around it could be clearer though.

Reply:
I am inclined to agree. We can’t convey precisely an unambiguous meaning with message name alone. Users will usually need to double check the doc for further clarity.

Reply:
I think Daniël probably had this in mind also.